lib/ui/utils/_utils.mjs

```diff
@@ -48,10 +48,13 @@ import imagePreview from './imagePreview.mjs'
 
 import resizeHandler from './resizeHandler.mjs'
 
+import json from './jsonDataview.mjs'
+
 export default {
   Chart,
   Tabulator,
   idleLogout,
   imagePreview,
+  json,
```

Comment:
Why is this called json and not jsonDataview?

Reply:
Agree. Should be jsonDataview.

Reply:
I have updated the reference.